examples/cmake_subdir/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.18.0)
+set(example pressio-example)
+project(${example})
+
+add_subdirectory(${${example}_SOURCE_DIR}/../.. ${${example}_BINARY_DIR}/pressio)
+
+add_executable(${example} ${CMAKE_CURRENT_SOURCE_DIR}/../main.cpp)
+target_link_libraries(${example} PUBLIC pressio)
+target_compile_definitions(${example} PUBLIC PRESSIO_ENABLE_TPL_EIGEN)
```

Comment:
can you also add the example where pressio headers are just included (via include_directories)?

Reply:
i am guessing that this is similar to the incldue usecase

Reply:
i still would like the "examples" to be moved into a separate repo. these are not really examples just examples for how to use the library.

Reply:
`include_directories` example:
https://github.com/Pressio/pressio-templates-usage-as-library/blob/main/cmake_include_directories/CMakeLists.txt